terraform/lib/dependabot/terraform/file_fetcher.rb

```diff
@@ -45,6 +46,13 @@ def terragrunt_files
           select { |f| f.type == "file" && terragrunt_file?(f.name) }.
           map { |f| fetch_file_from_host(f.name) }
       end
+
+      def lock_file
+        @lock_file ||=
```

Comment:
Looks like this is returning an array
```suggestion
      def lock_files
        @lock_files ||=
```